src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -2234,4 +2240,28 @@ public boolean isComplete() {
         }
         return instructorEmailNameTable;
     }
+
+    public StringBuilder getCsvDetailedFeedbackResponseCommentsRow(FeedbackResponseAttributes response) {
+        List<FeedbackResponseCommentAttributes> frcList = this.responseComments.get(response.getId());
+        StringBuilder commentRow = new StringBuilder(200);
+        for (FeedbackResponseCommentAttributes frc : frcList) {
+            commentRow.append("," + instructorEmailNameTable.get(frc.giverEmail) + ","
+                    + SanitizationHelper.desanitizeFromHtml(getTextFromComment(frc.commentText)));
```

Comment:
I believe desanitization is not required here anymore, since Jsoup will return desanitized text.